docs/transparent-validation/rte/v1.1.0/serial-port-connection-guide.md

```diff
@@ -0,0 +1,43 @@
+# Serial port connection guide
+
+This document describes how to set up a serial connection between RTE and
+another device.
+
+## Prerequisites
+
+* [Prepared RTE](../v1.1.0/quick-start-guide.md)
+* Two jumpers
+* Choose one of the following cables depending on the connectivity to your
+    platform:
+    - RS232-RJ45 cable
+    - RS232-RS232 cable
+    - 3 wire cables
+
+## Setting serial connection
+
+1. Connect the RTE header (J14 or J18) with the device serial port.
```

Comment:
In my opinion, the first point in this section should describe the determination method for connector type:
1. DUT has a serial port -> needs to use an RS232-RS232 cable.
2. DUT has the debug port in the form of the RJ45 -> needs to use an RS-232-RJ45 cable.
3. DUT does not have port, but have pins for serial connection on the board -> need to use a 3-wire cable.

Reply:
added